test/support/generator/random-value-generator.ts

```diff
@@ -33,11 +40,22 @@ export class RandomValueGenerator {
     return this.chance.floating({ min: 0, max: 1 });
   }
 
-  nonnegativeFloat(): number {
-    return this.chance.floating({ min: 0 });
+  nonnegativeFloat(fixed = 2): number {
+    return this.chance.floating({ min: 0, fixed });
   }
 
   date(): Date {
     return this.chance.date();
   }
+
+  // UKEF id example 0030000321. It should be used for Deal and Facility IDs.
+  // TODO: stringOfNumericCharacters should generate 6 digits, but doesn't support Max value at the moment.
+  ukefId(): UkefId {
+    return ('0030' + this.stringOfNumericCharacters(6)) as UkefId;
```

Comment:
Should we pick `0030` from constant please, since it will be utilised in future PRs as well?

Reply:
Changed to constant.
Changed from min 6 to length 6, as it was introduced in other PR. Removed TODO.